RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -158,21 +167,41 @@ private static void EnsureDummyViewEngineRegistered()
             }
         }
         
-        class DummyViewEngine : IViewEngine
+        public class DummyViewEngine : IViewEngine
```

Comment:
Move this out to a separate file if the plan is for users to be able to call in to it.